js/main.js

```diff
@@ -1,22 +1,109 @@
-function getRandomIntInclusive(min, max) {
-  if (min > max || min < 0 || max <= 0) {
-    return ('Неверный диапазон значений');
-  }
-  min = Math.ceil(min);
-  max = Math.floor(max);
-  return Math.floor(Math.random() * (max - min + 1)) + min;
-}
-getRandomIntInclusive (0, 87);
+const AVATAR_PHOTOS = [
+  'img/avatars/user{{01}}.png',
+  'img/avatars/user{{02}}.png',
+  'img/avatars/user{{03}}.png',
+  'img/avatars/user{{04}}.png',
+  'img/avatars/user{{05}}.png',
+  'img/avatars/user{{06}}.png',
+  'img/avatars/user{{07}}.png',
+  'img/avatars/user{{08}}.png',
+  'img/avatars/user{{09}}.png',
+  'img/avatars/user{{10}}.png'
```

Comment:
Массив можно генерировать по шаблону, например 
```suggestion
Array.from({length:10}, (_, index) => `img/avatars/user${(index + 1).toString().padStart(2, 0)}.png`)
```